pkgs/by-name/py/pyfa/package.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  appimageTools,
+  fetchurl,
+}:
+let
+  pname = "pyfa";
+  version = "2.60.0";
+  src = fetchurl {
+    url = "https://github.com/pyfa-org/Pyfa/releases/download/v${version}/pyfa-v${version}-linux.AppImage";
+    name = "pyfa-${version}";
+    sha256 = "sha256-bPInlYtsHtLk3AkF8Bf9CgJUAX7zRvdb6qO11MzF3eE=";
```

Comment:
We should use hash

Reply:
How would I change this to use hash instead?

Reply:
Just replace the sha256 attribute key to hash.

So "hash ="